x/cheqd/types/diddoc_verification_method.go

```diff
@@ -67,22 +55,29 @@ func VerifySignature(vm VerificationMethod, message []byte, signature []byte) er
 	var verificationError error
 
 	switch vm.Type {
-	case Ed25519VerificationKey2020:
-		_, keyBytes, err := multibase.Decode(vm.PublicKeyMultibase)
+	case Ed25519VerificationKey2020{}.Type():
+		var ed25519VerificationKey2020 Ed25519VerificationKey2020
```

Comment:
Validation and signature verification should belong to verification materials.

Reply:
@askolesov Makes sense.